go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/klauspost/compress v1.11.13
 	github.com/moby/locker v1.0.1
 	github.com/moby/sys/mountinfo v0.4.1
+	github.com/moby/sys/signal v0.5.0
```

Comment:
Note that I need to update this version before merge to keep all Windows signals as like before.